test/nyc-integration.js

```diff
@@ -1079,6 +1081,25 @@ describe('the nyc cli', function () {
     })
   })
 
+  it('help shows to stderr when main command doesn\'t know what to do', () => {
+    const opts = {
+      cwd: fixturesCLI,
+      env,
+      stripEof: false
+    }
+    return Promise.all([
+      execa(process.execPath, [bin, '--help'], opts),
```

Comment:
any reason we can't just use `spawnSync` here, I've been trying to use less dependencies when possible.

Reply:
execa is already a dependency (it's installed by yargs).  I'd like to leave the door open for parallel testing which can never happen if we use sync methods everywhere.

Reply:
^ agree with @coreyfarrell here